index.js

```diff
@@ -55,7 +55,7 @@ export default class TinySDF {
 
         // If the glyph overflows the canvas size, it will be clipped at the bottom/right
         const glyphWidth = Math.min(this.size - this.buffer, Math.ceil(actualBoundingBoxRight - actualBoundingBoxLeft));
-        const glyphHeight = Math.min(this.size - this.buffer, Math.ceil(actualBoundingBoxAscent + actualBoundingBoxDescent));
+        const glyphHeight = Math.min(this.size - this.buffer, Math.ceil(actualBoundingBoxAscent) + Math.ceil(actualBoundingBoxDescent));
```

Comment:
You mentioned that the same logic for `glyphWidth` "doesn't affect rendering": I think that's just because the fonts we're using typically have an `actualBoundingBoxLeft` that's slightly negative? So even if the resulting `glyphWidth` is one pixel too small, the glyph doesn't look clipped from the right because it's actually eating slightly into the buffer on the left.

Right now we're counting on that as a common property of fonts, since we hardwire the x drawing position to `this.buffer`, and hardwire the `left` metric to `0`. If we're worried, it would be straightforward to hook it up to the actual metrics the way we do for height.